### ISSUE TITLE
Cover image retieval doesn't work when the comic contains only GIF images (impact OPDS with fetch MediaInfo, file infos, ...)

### ISSUE THREAD

**Issue by foux** [mylar3/mylar3#663]
**Describe the bug**
When browsing the OPDS server, on some series I've got this error : 

```
11-Mar-2021 15:20:56 - ERROR :: mylar.extract_image.161 : CP Server Thread-3 : [ERROR] Unable to properly retrieve the cover. It's probably best to re-tag this file : local variable 'low_infile_name' referenced before assignment
[11/Mar/2021:15:20:56] HTTP
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/cherrypy/_cprequest.py", line 638, in respond
    self._do_respond(path_info)
  File "/usr/lib/python3.8/site-packages/cherrypy/_cprequest.py", line 697, in _do_respond
    response.body = self.handler()
  File "/usr/lib/python3.8/site-packages/cherrypy/lib/encoding.py", line 219, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/cherrypy/_cpdispatch.py", line 54, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "/app/mylar3/mylar/webserve.py", line 5885, in opds
    data = op.fetchData()
  File "/app/mylar3/mylar/opds.py", line 87, in fetchData
    result = methodToCall(**self.kwargs)
  File "/app/mylar3/mylar/opds.py", line 404, in _Comic
    issuedetails = mylar.helpers.IssueDetails(fileloc).get('metadata', None)
  File "/app/mylar3/mylar/helpers.py", line 1533, in IssueDetails
    IssueImage = file_info['ComicImage']
TypeError: 'NoneType' object is not subscriptable
```

This appears on multiple distinct series.

**CarePackage**

[carepackage.zip](https://github.com/mylar3/mylar3/files/6123638/carepackage.zip)

**Environment (please complete the following information):**
 - OS: Linux/Docker
 - Branch / commit: Version: v0.4.9.6 ( master)

**Comment by evilhero** [comment]
Have you tried to re-tag one of the files that it says it's having problems with, like it mentions in the log? 

It's possible it might be a conversion problem of some kind due to encoding, albeit I don't know off-hand what it would be since the I can't really tell much from the carepackage since it's all at an INFO level of logging. 

If retagging doesn't help, you'd have to put Mylar into debug/verbose mode and then perform the task that's causing the error so it'll get fully logged, and then attach that carepackage.

**Comment by foux** [comment]
Thansk for your quick reply @evilhero ! A retagging didn't help. Here is the care package in debug mode : 
[carepackage.zip](https://github.com/mylar3/mylar3/files/6124754/carepackage.1.zip)

**Comment by foux** [comment]
Disabling `OPDS Fetch MetaInfo` allows the OPDS to work for those comics

**Comment by evilhero** [comment]
I'm betting that the .jpgs inside of the cbz are all named something that the parser can't figure out what is the 'cover'.

If, in Mylar, you go to the series for an issue that's having problems, and click on the blue ```i``` beside the given issue - does it display in the GUI or does it throw back an error as well ?

Can you verify the naming convention of the jpgs inside the cbz - or better yet, throw one of the files up somewhere that is given you problems so that I can run some tests against it ?

**Comment by foux** [comment]
I'll have to check all of the files since I don't know which one is firing the issue. I'll let you know!

**Comment by foux** [comment]
Well, the issues is with all files, and the `i` throws an error too.

Here is one of the files with the issue : 
[Lefranc 6 (1978).zip](https://github.com/mylar3/mylar3/files/6125446/Lefranc.6.1978.zip)

**Comment by evilhero** [comment]
The problem is that the images inside are all in .gif format - Mylar only looks for .jpg, .png, and .webp.

I tried quickly to patch it up, but I can't get the gif to display within Mylar as it needs to resize it to display it, which brings into a whole whack of img stuff (I got it to display without an error, but it won't be able to display any gifs so the OPDS wouldn't show anything for pages either).

I'm not sure about how to patch this up if I'm being honest - I've got my hands in too many other things right now to go delving into image formats and conversions, but I'll leave this open in case someone else wants to try their hand at patching it up to display the format correctly, or for when there's more time for me to go through it.

**Comment by foux** [comment]
It's honnestly not a deal killer for me, as we can use it when disabling the Fetch MediaInfo!
Congrats on finding the source of the issue.

**Comment by barbequesauce** [comment]
Hopefully this will fix it @foux, stay tuned

**Comment by barbequesauce** [comment]
Should have been fixed in faa6ee68f989b03332147fa9e29bfc2d97c4a3bd - please update & confirm
